src/index.html

```diff
@@ -9,10 +9,45 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/index.css"
```

Comment:
The link to the stylesheet should use a relative path starting with './' to indicate that the file is in a subdirectory relative to the current file. Change `href="styles/index.css"` to `href="./styles/index.css"`.